### PR TITLE
Blog App: Final work

### PR DESCRIPTION
**Pull request by amare1990**
- [ ] Setup rails and other necessary tools.
- [ ] Created models and setup associations between models.
- [ ] Added model validations.
- [ ] Used `devise` authentication tool to create user registrations and login.
- [ ] Used 'cancancan` gem library to create ability authorization file.
- [ ] Created categories and transaction `#index`, `#show`, `#new` pages that sticks to the design given
- [ ] Added unit tests and integration tests for models, controllers and views.
- [ ] Added CSS styles for each page.
- [ ] Fixed linters errors.
- [ ] Updated README.md microverse template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,231 +22,178 @@ After you're finished please remove all the comments and instructions!
...
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [video Demo](https://loom.com/share/c91c330b800d41b89d88e5d84b86cf5e)
+
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://amare-budget-mgt-app.onrender.com/)
```
Comment (danielufeli) [inline README.md]: - [ ] Nice Deploying your project online, kindly check the page is showing a `502` error.
502_error

File: app/views/splash_screen/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="splash-screen-container">
```
Comment (danielufeli) [inline app/views/splash_screen/show.html.erb:1]: - [ ] Great work done on the splash screen, kindly align the text in the buttons so they are well aligned following the design guide. See screenshot below

### Your Design
splash_screen

### Design Guide

usable

File: spec/views/categories/edit.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'categories/edit', type: :view do
+  let(:category) do
+    Category.create!(
```
Comment (danielufeli) [inline spec/views/categories/edit.html.erb_spec.rb]: - [ ] Good effort testing your application, I resolved the error with the browser using [this article](https://stackoverflow.com/questions/69316284/webdriversbrowsernotfound-failed-to-find-chrome-binary) on stack overflow by installing this package `sudo apt-get install chromium-chromedriver`, kindly check some tests are still failing see screenshot below.

test_fall

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @amare1990 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
- Nice commits messages✔️
- Clean Code✔️

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/views/categories/edit.html.erb_spec.rb (hunk 3 above)
Comment (amare1990) [reply spec/views/categories/edit.html.erb_spec.rb]: Thanks @danielufeli  for carefully reviewing my project. I really appreciate you for giving me references to the challenges I have been experiencing on my platform (Linux OS). Hopefully I will fix the problems and make changes. Thanks again.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @amare1990, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did well to implement the changes requested by the previous reviewer.

## Optional suggestions

- [ ] We should always follow the correct workflows when working with GitHub, in this project you were required to use `gitflow` and not `github flow`. In this case, a feature branch would have been created from the development branch  and request a review from there.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**